packages/vlang/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://vlang.io/
+TERMUX_PKG_DESCRIPTION="V programming language compiler"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@ilyas-ozt"
+TERMUX_PKG_VERSION="0.1.0"
+TERMUX_PKG_GIT_URL=https://github.com/vlang/v
+TERMUX_PKG_GIT_BRANCH=master
```

Comment:
We probably don't need or want to build this package from a git clone.

We usually use the release tarballs, this has the main benefit that we don't need to pull down the entire git repo, and it lets us easily verify the integrity of the source archive via a checksum.